### DISCUSSION TITLE
没有找到CreationBench数据集

### DISCUSSION THREAD

**Discussion by zhangfelix** [open-compass/opencompass#1117] [Q&A] [answered]
我看到commit里面有添加CreationBench数据集，测评脚本里面也有相关内容，但是OpenCompassData-core-20240207.zip和OpenCompassData-complete-20240325.zip都没有找到相关数据集。

**Comment by bittersweet1999** [comment, marked as answer]
这个数据集是我们自己构建的，在整理公开中
